main_panel.py

```diff
@@ -58,11 +61,13 @@
 
 #-------------------------------------------------------------------------------------
 
+child_ages = string_to_list_of_integers(child_age_input.value)
```

Comment:
this needs to be inside the update function, so that the latest value is used when the Calculate button is pushed